vm-setup/roles/v1aX_integration_test/tasks/move.yml

```diff
@@ -0,0 +1,90 @@
+---
+  - name: Label BMO CRDs.
+    shell: 'kubectl label --overwrite crds baremetalhosts.metal3.io {{ item }} --overwrite '
+    with_items:
+       - clusterctl.cluster.x-k8s.io=""
+       - cluster.x-k8s.io/provider="metal3"
+    
+  - name: Obtain target cluster kubeconfig
+    shell: "kubectl get secrets {{ CLUSTER_NAME }}-kubeconfig -n {{ NAMESPACE }} -o json | jq -r '.data.value'| base64 -d > /tmp/target.yaml"
+    register: target_kubeconfig
```

Comment:
Why do you register this ? it seems unused

Reply:
It is unused indeed. I can remove it later

Reply:
Done